### PR TITLE
HOMS-272 Migrate from thin to unicorn

### PR DESCRIPTION
**Pull request by TimAle**

### CODE REVIEW

File: config/unicorn.rb [hunk 1]
```diff
@@ -1,44 +1,64 @@
-app_root = '/var/www/homs'
-rails_root = "#{app_root}/current"
+rails_root = '/opt/homs'
 
 working_directory rails_root
-worker_processes 4
+worker_processes  ENV['HOMS_UNICORN_WORKERS'].to_i || 16
```
Comment (sugarfree1) [inline config/unicorn.rb]: Actually we reduced number of workers to 8.

File: .env.sample [hunk 2]
```diff
@@ -6,6 +6,9 @@ MINIO_BUCKET_NAME=bucket_name
 MINIO_ACCESS_KEY=minio_access_key
 MINIO_SECRET_KEY=minio_secret_key
 
+HOMS_UNICORN_WORKERS=16
```
Comment (sugarfree1) [inline .env.sample]: Let it be 8

File: Gemfile [hunk 3]
```diff
@@ -26,7 +26,6 @@ gem 'apitome'
 gem 'dry-container'
 gem 'dry-auto_inject'
 gem 'dry-validation'
-gem 'thin'
```
Comment (sugarfree1) [inline Gemfile]: thin is still ok for development and testing

Comment (TimAle) [reply Gemfile]: We can just leave it, but I thought we use WEBrick in these envs.

Comment (sugarfree1) [reply Gemfile]: ah, yes
okay, let it be

**Pull request merged**